src/components/scenes/ConfirmScene.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react'
+import { View } from 'react-native'
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view'
+
+import { useHandler } from '../../hooks/useHandler'
+import s from '../../locales/strings'
+import { NavigationProp, RouteProp } from '../../types/routerTypes'
+import { SceneWrapper } from '../common/SceneWrapper'
+import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
+import { EdgeText } from '../themed/EdgeText'
+import { MainButton } from '../themed/MainButton'
+import { SafeSlider } from '../themed/SafeSlider'
+import { SceneHeader } from '../themed/SceneHeader'
+import { Tile } from '../tiles/Tile'
+interface Props {
+  navigation: NavigationProp<'confirmScene'>
+  route: RouteProp<'confirmScene'>
+}
+
+export interface ConfirmSceneParams {
+  titleText: string
+  bodyText: string
+  infoTiles?: Array<{ label: string; value: string }>
+  onConfirm: (resetSlider: () => void) => void
+  onBack?: () => void
+}
+
+const ConfirmComponent = (props: Props) => {
+  const { navigation, route } = props
+  const theme = useTheme()
+  const styles = getStyles(theme)
+
+  const { titleText, bodyText, infoTiles, onConfirm, onBack } = route.params
+
+  const renderInfoTiles = () => {
+    if (!infoTiles || !infoTiles.length) return null
```

Comment:
We don't have the lint rule turned on yet, but you have to be explicit with your boolean checks. `if (infoTiles == null) return null`. You don't need to worry about length === 0, since the empty array is no problem for React.

Reply:
Ah yes, this was a super copy-pasta bug. Came from old SendScene -> SendScene2 -> ConfirmScene but I should have been better. Will fix